doc/cabal-package-description-file.rst

```diff
@@ -1356,8 +1357,7 @@ A typical stanza for a foreign library looks like
    have any dependencies on other (Haskell) shared libraries; without
    the ``standalone`` option the generated library would have dependencies
    on the Haskell runtime library (``libHSrts``), the base library
-   (``libHSbase``), etc. Currently, ``standalone`` *must* be used on Windows
-   and *must not* be used on any other platform.
+   (``libHSbase``), etc. The ``standalone`` option *must* be used on Windows.
```

Comment:
Still does not say _why_ this must be used on windows. I mean sure, we do _not_ have dynamic linking on window (for Haskell libraries) at all! So if you _do not_ pass `standalone` you end up with only the library you are building, but it lacks all the dependencies. (I think).

Reply:
It does not say why because I did not know why. What you say sounds reasonable, perhaps we should do some tests.